cocotb/share/lib/embed/gpi_embed.cpp

```diff
@@ -258,11 +258,6 @@ extern "C" int embed_sim_init(int argc, char const * const * argv)
         LOG_ERROR("Failed to get the _log_from_c function");
         goto cleanup;
     }
-    if (!PyCallable_Check(simlog_func)) {
-        LOG_ERROR("_log_from_c is not callable");
-        Py_DECREF(simlog_func);
-        goto cleanup;
-    }
```

Comment:
Just a note - this is safe anyway, as if it's not callable we'll get a `TypeError` later, which isn't any different really from the function itself failing.